### PR TITLE
ZK-4180: Remove IE < 9 workarounds

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zk/src/archive/web/js/zk/dom.js [hunk 1]
```diff
@@ -1084,7 +1083,7 @@ jq(el).zk.sumStyles("lr", jq.paddings);
 	toStyleOffset: function (x, y) {
 		var el = this.jq[0],
 			oldx = el.style.left, oldy = el.style.top,
-			resetFirst = zk.webkit || zk.opera || zk.air || zk.ie > 7; // don't use zk.ie8 which is not including ie 11
+			resetFirst = zk.webkit || zk.opera || zk.air || zk.ie > 8; // don't use zk.ie9 which is not including ie 11
```
Comment (plijyfes) [inline zk/src/archive/web/js/zk/dom.js]: >= 9

File: zul/src/archive/web/js/zul/sel/Select.js [hunk 2]
```diff
@@ -308,11 +308,6 @@ zul.sel.Select = zk.$extends(zul.Widget, {
 
 		this.fire('onSelect', {items: data, reference: reference});
 	},
-	//Bug 3304408: IE does not fire onchange
-	doBlur_: zk.ie8_ ? function (evt) { // only apply to IE 8
-		this._doChange(evt);
-		return this.$supers('doBlur_', arguments);
-	} : zk.$void,
```
Comment (plijyfes) [inline zul/src/archive/web/js/zul/sel/Select.js]: keep zk.$void

File: zk/src/archive/web/js/zk/zk.js [hunk 3]
```diff
@@ -353,27 +353,6 @@ zk.copy(zk, {
 	 * @since 5.0.7
 	 */
 	//iex: null,
```
Comment (scribetw) [inline zk/src/archive/web/js/zk/zk.js]: confirm iex

File: zul/src/org/zkoss/zul/impl/PageRenderer.java [hunk 4]
```diff
@@ -69,9 +69,6 @@ protected void renderDesktop(Execution exec, Page page, Writer out) throws IOExc
 		out.write("<html");
 		if (number == null || number.intValue() == 0) {
 			Double ie = exec.getBrowser("ie");
```
Comment (scribetw) [inline zul/src/org/zkoss/zul/impl/PageRenderer.java]: remove ie

File: zweb/src/org/zkoss/web/fn/ThemeFns.java [hunk 5]
```diff
@@ -115,8 +115,7 @@ public static String gradient(String direction, String colors) {
 		else {
 			String[] cols = colors.split(";");
 			StringBuilder sb = new StringBuilder();
-			if (temp != Browser.Old_IE)
```
Comment (scribetw) [inline zweb/src/org/zkoss/web/fn/ThemeFns.java]: Remove Old_IE.
`IE <= 9` = IE9

**Pull request merged**